test/SpecRunner.html

```diff
@@ -24,6 +25,7 @@
 
   <!-- include source files here... -->
   <script src="../../fast-json-patch/src/json-patch-duplex.js"></script>
+  <script src="../../../node_modules/jsonpatcherproxy/dist/jsonpatcherproxy.min.js"></script>
   <script src="../src/palindrom.js"></script>
   <script src="../src/palindrom-dom.js"></script>
 
```

Comment:
But are they devDependencies? Since these files are installed using NPM, what the point of installing them again using Bower?